_data/nav.yml

```diff
@@ -162,7 +162,8 @@ About:
   - href: /about/
   - href: /about/key-benefits/
   - href: /about/product-values/
-  - href: /about/contribute/
+  - text: Contribute
+    href: /about/contribute/
```

Comment:
I created this so that the sidenav link could stay as "Contribute". But it might not be necessary! Happy to remove if we are ok with the sidenav saying "Contribute to USWDS"

Reply:
@amyleadem I see, I wasn't aware we wanted to keep the title as `Contribute to USWDS`. I was suggesting changing the page title because it'd be consistent everywhere.

In that case, we can ignore this.